src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -1,10 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.IL">
+  <!-- Make sure that the DebuggableAttribute is set properly. -->
+  <PropertyGroup>
+    <DebugOptimization>IMPL</DebugOptimization>
+    <DebugOptimization Condition="'$(ConfigurationGroup)' == 'Release'">OPT</DebugOptimization>
```

Comment:
Is there a downside to just always using OPT for this project?

Reply:
I'm not 100% sure.  I was attempting to match what we see for debug and release builds on other binaries in an attempt to not make this one special.

Reply:
Ok